launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/EnvCloudScaffolder.kt

```diff
@@ -133,30 +110,44 @@ class EnvCloudScaffolder(private val launcher: Launcher) {
         )
     }
 
+    @Suppress("LongMethod", "MaxLineLength")
     private fun saveEnvSrcFiles() {
         launcher.workFileOnce("env/src/environment/docker-compose.yml.peb") {
             println("Saving environment Docker compose file '$this'")
             writeText(
                 """
                 version: "3"
                 services:
-                  httpd:
-                    image: centos/httpd:latest
-                    command: ["tail", "-f", "--retry", "/usr/local/apache2/logs/error.log"]
+                  dispatcher:
+                    image: adobe/aem-ethos/dispatcher-publish:latest
```

Comment:
That's wrong. Please use {{ dispatcherImage}}  variable

Reply:
done